pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1911,7 +1912,7 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
         await Promise.all([
           this.progressTracker.action('Installing CA certificates', 50, this.installCACerts()),
           this.progressTracker.action('Configuring image proxy', 50, this.configureOpenResty(config)),
-          this.progressTracker.action('Configuring containerd', 50, this.configureContainerd()),
+          this.progressTracker.action('Configuring container engine', 50, this.configureContainerd()),
```

Comment:
Hmm, we run `configureContainerd()` regardless of engine selection, don't we. It might be good to renamed the method while we're at it…

Reply:
Renamed to `configureContainerEngine`, which is kind of confusing because we also have `BackendHelper.configureContainerEngine`, but the latter is called by the former, so maybe it is all good.